src/app/series/directives/series-podcast.component.ts

```diff
@@ -101,4 +101,21 @@ export class SeriesPodcastComponent implements OnDestroy, DoCheck {
     }
   }
 
+  resetNewFeedUrlOnCancel(confirm) {
```

Comment:
At first I found the "Cancel == revert change" thing a bit confusing.  But calling that button "Discard" would be even more confusing, since it just discards the one field, not the whole form.  So I think I'm good with this.